tm2/pkg/bft/consensus/config/config.go

```diff
@@ -20,6 +20,8 @@ type ConsensusConfig struct {
 	WalPath string `toml:"wal_file"`
 	walFile string // overrides WalPath if set
 
+	WALDisabled bool `toml:"wal_disabled" comment:"Disable the write-ahead log"`
```

Comment:
I'd make this consistent with `WalPath` (suggest making that one `WALPath` following the convention); maybe place this next to `WalPath` in the struct?

Also I'm not sure whether this should be exposed in the config.

Reply:
I've done what you suggested and removed the toml tag